config/application.yml.example

```diff
@@ -33,6 +33,7 @@ development:
   session_timeout_in: '15'
   telephony_disabled: 'true'
   twilio_accounts: '[{"sid":"sid", "auth_token":"token", "number":"9999999999"}]'
+  valid_service_providers: '["https://rp1.serviceprovider.com/auth/saml/metadata", "urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost", "urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost-rails", "https://dashboard.login.gov"]'
```

Comment:
can this use the yaml style:

```yaml
valid_service_providers:
  - 'some_host'
  - 'some_other_host'
```

instead? easier to read, imo, and will scale up better as we add agencies.